src/modules/schoolingOverview/docs/en_US.md

```diff
@@ -0,0 +1,21 @@
+Especially in large associations, the list of available courses quickly becomes very long. If you take part in many training courses even then, you will quickly lose track of what's going on.
```

Comment:
Do they also call it "course" ingane or do they use the term "schooling"?
Same thing in the translation file(s) 😉

Reply:
No is Courses :D https://cdn.discordapp.com/attachments/361252906806018068/769462632113242172/unknown.png